modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro

```diff
@@ -0,0 +1,130 @@
+definition {
+
+	macro _formatCamelCase {
+		var camelCase = StringUtil.replace("${camelCase}", " ", "");
+
+		var camelCase = StringUtil.lowerCaseFirstLetter("${camelCase}");
+
+		return "${camelCase}";
+	}
+
+	macro assertLincenseQuantity {
```

Comment:
@MathausC would you mind to check the word "Lincense" here. There is a typo, it should by "License". Thank you!